cluster/sync.sh

```diff
@@ -49,7 +49,7 @@ REGISTRY=$manifest_registry make manifests
 until [[ $(./cluster/kubectl.sh get --ignore-not-found -f $bridge_marker_manifest 2>&1 | wc -l) -eq 0 ]]; do sleep 1; done
 until [[ $(./cluster/kubectl.sh get --ignore-not-found ds bridge-marker 2>&1 | wc -l) -eq 0 ]]; do sleep 1; done
 
-./cluster/kubectl.sh create -f $bridge_marker_manifest
+sed 's/quay.io\/kubevirt/registry:5000/g' examples/bridge-marker.yml | ./cluster/kubectl.sh create -f -
```

Comment:
Why is this not using the bridge_marker_manifest variable?

Reply:
No reason. Done.